rtengine/improccoordinator.cc

```diff
@@ -1286,6 +1286,10 @@ void ImProcCoordinator::updatePreviewImage(int todo, bool panningRelatedChange)
             nprevl->CopyFrom(oprevl);
             histCCurve.clear();
             histLCurve.clear();
+            bool hasColorToningLabGrid = params->colorToning.enabled && params->colorToning.method == "LabGrid";
```

Comment:
See previous comment. No variable needed imo.

Reply:
Yes we can remove, I keep only for **visibility**..with "old" version.
This change is in - dcrop, improccoordinator, rtthumbnail, simpleprocess

You can modified and push this chnage if you want :)


Jacques